### ISSUE TITLE
Feature Request - Password Support

### ISSUE THREAD

**Issue by TheRealFlamepanther** [blzla/SGB-Settings-Editor#2]
SGB2 (and possibly SGB1--I haven't looked in some time) provides passwords for custom user-created palettes in its palette editor. Would it be possible to add support for loading these into the actively selected palette in the Settings Editor? That way, users could create or adjust their palettes in-game on the SGB itself and then bring them over to store them in the ROM.

**Comment by blzla** [comment]
Hey, this has been requested before and I'm currently working on it. I've tried to reverse engineer the algorithm from the game, but it takes 7000 instructions from reading the number to arriving at the color, so I gave up. I then decided to go for a lookup table (LUT) approach, with the help of another person, but after checking the output, I noticed that there were many errors in their data.

I currently have a script that logs the colors automatically from the rom, so I should get this feature done in the next days. If anyone wants to reverse the actual algorithm, feel free to contribute.
